taskvine/src/manager/vine_manager.c

```diff
@@ -4430,8 +4458,11 @@ const char *vine_result_string(vine_result_t result)
 	case VINE_RESULT_CANCELLED:
 		str = "CANCELLED";
 		break;
-	case VINE_RESULT_LIBRARY_EXIT:
-		str = "LIBRARY_EXIT";
+	case VINE_RESULT_LIBRARY_CANCELLED:
```

Comment:
Hmm, I don't think that we want to introduce too many specialized states here.
- `LIBRARY_CANCELLED` should be the same thing as `CANCELLED` (since it gets cancelled in the same way and we want to count up statistics in the same way.
- In the same way `LIBRARY_FAILED` is unneeded b/c anytime a library task runs to completion, we should consider it a failure.